### PR TITLE
Added "Reuse last value" section

### PR DESCRIPTION
**Pull request by tomasMizera**
Added section for feature `Reuse last entered values` - `/howto/reuse_entered_values` 

Merge only after this PR is merged / released https://github.com/lutraconsulting/input/pull/1115

@saberraz or @mostlyAtNight feel free to correct any mistakes.

### CODE REVIEW

File: www/pages/reuse_last_values.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: page
+permalink: /howto/reuse_entered_values
+---
+
+<!--- IMPORTANT: This permlink is referenced from InputApp -->
```
Comment (wonder-sk) [inline www/pages/reuse_last_values.md:6]: I don't think it is referenced directly from input

Comment (tomasMizera) [reply www/pages/reuse_last_values.md:6]: True, it is not. That was a copy&paste. However, shall we add it somewhere to the app?

Comment (wonder-sk) [reply www/pages/reuse_last_values.md:6]: In theory we could have a "learn more" link in the Settings near that option... but from the time being I am happy to leave it as is and just remove the misleading comment

Comment (tomasMizera) [reply www/pages/reuse_last_values.md]: Sure, done.

**Pull request merged**